src/python/tests/fasta/test_chunk.py

```diff
@@ -12,12 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Unit testing of `ensembl.io.genomio.fasta.chunk` module.
-
-Typical usage example::
-    $ pytest test_chunk.py
-
-"""
+"""Unit testing of `ensembl.io.genomio.fasta.chunk` module."""
+# pylint: disable=too-many-positional-arguments
```

Comment:
Slightly confused on adding this comment line, does this disable TMPA ?

Reply:
Obvs same applies to other insertions of this line, just to clarify thanks

Reply:
Yes, this disables that particular check for the whole file. I would have preferred to disable it completely for the `tests` folder, but `pylint` does not have such feature, so it has to be manually introduced in each test file.

Reply:
Oh ok ok just checking as I wasn't sure if that enabled or disabelled the check thanks for clearing that up. Merge approved from my end